docs/en_US/Tutorial/QuickStart.rst

```diff
@@ -227,80 +227,43 @@ After you start your experiment in NNI successfully, you can find a message in t
 
 Open the ``Web UI url`` (Here it's: ``[Your IP]:8080``\ ) in your browser; you can view detailed information about the experiment and all the submitted trial jobs as shown below. If you cannot open the WebUI link in your terminal, please refer to the `FAQ <FAQ.rst>`__.
 
-View summary page
-^^^^^^^^^^^^^^^^^
+View overview page
+^^^^^^^^^^^^^^^^^^
 
-Click the "Overview" tab.
 
-Information about this experiment will be shown in the WebUI, including the experiment trial profile and search space message. NNI also supports downloading this information and the parameters through the **Download** button. You can download the experiment results anytime while the experiment is running, or you can wait until the end of the execution, etc.
+Information about this experiment will be shown in the WebUI, including the experiment trial profile and search space message. NNI also supports downloading this information and the parameters through the **Experiment summary** button.
 
 
-.. image:: ../../img/QuickStart1.png
-   :target: ../../img/QuickStart1.png
-   :alt: 
+.. image:: ../../img/webui-img/full-oview.png
+   :target: ../../img/webui-img/full-oview.png
+   :alt: overview
```

Comment:
better to delete unused pictures

Reply:
`quickStart1.png` and `quickStart2.png` and `quickStart3.png` ... `quickStart7.png` had been deleted